src/Microsoft.AspNetCore.Server.IIS/WebHostBuilderIISExtensions.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.AspNetCore.Hosting.Server;
+using Microsoft.AspNetCore.Server.IIS;
+using Microsoft.AspNetCore.Server.IIS.Core;
+
+namespace Microsoft.AspNetCore.Hosting
+{
+    public static class WebHostBuilderIISExtensions
+    {
+        /// <summary>
+        /// Configures the port and base path the server should listen on when running behind AspNetCoreModule.
+        /// The app will also be configured to capture startup errors.
+        /// </summary>
+        /// <param name="hostBuilder"></param>
+        /// <returns></returns>
+        public static IWebHostBuilder UseIIS(this IWebHostBuilder hostBuilder)
```

Comment:
UseIISServer?

Reply:
We don't have `Server` suffix in existing methods: `UseKestrel`, `UseHttpSys`

Reply:
UseIISInProcess()? UseIIS is ambiguous to me. cc @shirhatti

Reply:
I'd say `UseIIS` to be consistent.

That being said, I'm not a fan of the prior art. `UseHttpSys` is even more ambiguous name.

Reply:
From a customer standpoint, it will be hard to know the difference between UseIISIntegration and UseIIS.

Reply:
But I'm hard pressed to see how UseIISIntegration vs UseIISServer is easier for customers

Reply:
I vote UseIIS, it's confusing regardless.